### PR TITLE
Fix runtime error when parsing meshes due to using float in range loop

### PR DESCRIPTION
**Pull request by astumpf**
Simple fix of runtime error when parsing meshes.

### CODE REVIEW

**Comment by omichel** [comment]
Do you have an example of a mesh which was causing trouble here? Normally the length of the texture coordinate index list should always be a multiple of 3, otherwise, your mesh may be corrupted.

**Comment by astumpf** [comment]
It is indeed always 3, but division returns a `float` type, which cannot be used as `range`.

File: urdf2webots/writeRobot.py [hunk 1]
```diff
@@ -548,7 +548,7 @@ def URDFVisual(robotFile, visualNode, level, normal=False):
                         if len(value) == 3:
                             robotFile.write('%d %d %d -1 ' % (value[0], value[1], value[2]))
                 elif isinstance(visualNode.geometry.trimesh.texCoordIndex[0], np.int32):
-                    for i in range(len(visualNode.geometry.trimesh.texCoordIndex) / 3):
+                    for i in range(int(len(visualNode.geometry.trimesh.texCoordIndex) / 3)):
```
Comment (omichel) [inline urdf2webots/writeRobot.py:551]: I see, in that case, we could maybe use the `//` operator:
```suggestion
                    for i in range(len(visualNode.geometry.trimesh.texCoordIndex) // 3):
```

Comment (astumpf) [reply urdf2webots/writeRobot.py]: You're right. Wasn't aware of the integer division.

**Review by omichel — APPROVED** [review]

**Pull request merged**